pset-functions.js

```diff
@@ -30,7 +30,22 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+const playerCardScore = 19
+const dealerCardScore = 22
+
+
+function blackJack(x, y) {
+  if (y > 21 && x > 21) {
+    return 0
+} else if ( y <= 21 && y > x || y <= 21 && y !== x) {
+    return y
+} else if (x <= 21 && x > y || x <= 21 && x !== y  ) {
+    return x
+} else ( x <= 21 && y <= 21 && (x === y ))
+  return "This hand is a push."
+}
```

Comment:
Kristen, great use of if/else statements here; the logic is really well thought out and captures every possible scenario. A few suggestions to keep in mind: instead of declaring global variables on lines 33-34, you can use the function's parameters as placeholder variable names (example found [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L33)).